service/worker/scheduler/workflow.go

```diff
@@ -690,7 +676,8 @@ func (s *scheduler) addStart(nominalTime, actualTime time.Time, overlapPolicy en
 }
 
 // processBuffer should return true if there might be more work to do right now.
-func (s *scheduler) processBuffer() (bool, error) {
+//nolint:revive
```

Comment:
What does `revive` complain about here?

Reply:
Anyone is welcome to take a try at simplifying this, though